libgeopm/src/geopmctl_main.c

```diff
@@ -29,7 +28,7 @@ int main(int argc, char **argv)
 {
     int opt;
     int err0 = 0;
-    char error_str[NAME_MAX] = {0};
```

Comment:
I think there's a pre-existing bug in this file. Line 77 calls `geopm_error_message(err0, error_str, GEOPMCTL_STRING_LENGTH);`. Should this declaration use `GEOPMCTL_STRING_LENGTH` instead?